lzy/long-running/src/main/java/ai/lzy/longrunning/task/DispatchingOperationTaskResolver.java

```diff
@@ -0,0 +1,56 @@
+package ai.lzy.longrunning.task;
+
+import ai.lzy.model.db.TransactionHandle;
+import com.google.common.annotations.VisibleForTesting;
+import jakarta.annotation.Nullable;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.sql.SQLException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DispatchingOperationTaskResolver implements OperationTaskResolver {
```

Comment:
Task resolver that can accept a list of different typed resolver to choose resolver by a task type.